src/itdelatrisu/opsu/Utils.java

```diff
@@ -756,4 +756,14 @@ public static long getUsedMemory() {
 		Runtime r = Runtime.getRuntime();
 		return r.totalMemory() - r.freeMemory();
 	}
+
+	public static class Pair<F, S> {
```

Comment:
Eh... not a fan of this abstraction.  It's barely more clear than just adding 2 elements in a row to your lists since this still doesn't tell you what they are, plus it's extra overhead.